plugins/action_grant_permissions/src/grantPermissions.ts

```diff
@@ -1,4 +1,7 @@
-const grantPermissions = async ({ username, orgName, permissionNames }: any, DBConnect: any) => {
+import { ActionPluginInput } from '../../types'
+
+const grantPermissions = async ({ parameters, DBConnect }: ActionPluginInput) => {
+  const { username, orgName, permissionNames } = parameters
```

Comment:
Similar to previous comment `orgName` can be either from `parameters` or `applicationData`.

Reply:
Actually, orgName is not currently available on `applicationData`, since every other plugin uses orgId. Do you think we should add it? (It'd require an additional database query in `getApplicationData.ts`).

Just need be a bit careful with inferring with this plugin, since `orgName` can legitimately be `null`, so would need to be quite careful making sure all cases work for this Action.

Reply:
No, it's ok to pass it through, I was just checking this agains the `List-of-Actions-plugins` doc. This needs to be updated there, so `orgName` is a required param

Reply:
It's not required, though -- you can grant a permission to a user without needing an Org. :)